SpatialGDKEditorCommandlet/SpatialGDKEditorCommandlet.uplugin

```diff
@@ -0,0 +1,30 @@
+{
+	"FileVersion" : 3,

```

Comment:
Equivalent sort of feeling here. Does this need to be a plugin or can it be a module?

Reply:
See above comments.